### ISSUE TITLE
[EagleMeshTransformer ] Question regarding training data

### ISSUE THREAD

**Issue by mijanr** [eagle-dataset/EagleMeshTransformer#8]
Hi, 

Great work, and thanks for publishing the source code. 

**So, my questions are:**
1. What kind of simulation data did you use for benchmarking? Is it some LES data or what?
2. And is it not the case that the kind of data I will train the model with should be the kind of data the model will predict? Which is, if I train the model with 100 trajectory of LES data, the model should generalize on the LES data for the specific case I am training with, and for the predictions on new trajectories, it should predict LES quality data, right?

I appreciate your response to my question. 
Thank you.

**Comment by eagle-dataset** [comment]
Hello,

- As mentionned in the paper, numerical simulations were carried out using the software Ansys Fluent to solve the
Reynolds Averaged Navier-Stokes equations with Reynolds stress mode.
- The model should indeed match the accuracy of the training data, and is certainly biased by it (a model trained on RANS simulation might perform less well if evaluated on similar simulation solved with LES). Unfortunately, I fear that 100 LES simulations won't be enough for allowing generalization though...

Thanks !
